### PR TITLE
Bridge sessions, threadpool, request timing and connection pool metrics to MP Telemetry + minimum FATS 

### PR DESCRIPTION
**Pull request by Channyboy**
#build

### CODE REVIEW

File: dev/com.ibm.ws.connectionpool.monitor/src/com/ibm/ws/connectionpool/monitor/metrics/MetricsManager.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Tr.debug(tc, "No MetricsManager Instance available ");
+        }
+        return null;
+    }
+
+    /**
+     *
+     * @param httpStatAttributes
```
Comment (fmhwong) [inline dev/com.ibm.ws.connectionpool.monitor/src/com/ibm/ws/connectionpool/monitor/metrics/MetricsManager.java]: The javadoc doesn't match the method parameters.

File: dev/com.ibm.ws.connectionpool.monitor/src/com/ibm/ws/connectionpool/monitor/metrics/MetricsManager.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+     * @param duration
+     * @param appName            The appName, can be null to indicate server context
+     */
+    public void updateWaitTimeMetrics(String poolName, Duration duration) {
+        metricRuntimes.stream().forEach(adapters -> adapters.updateWaitTimeMetrics(poolName, duration, getApplicationName()));
+    }
+
+    /**
```
Comment (fmhwong) [inline dev/com.ibm.ws.connectionpool.monitor/src/com/ibm/ws/connectionpool/monitor/metrics/MetricsManager.java:81]: The javadoc doesn't match the method parameters.

File: dev/com.ibm.ws.connectionpool.monitor/src/com/ibm/ws/connectionpool/monitor/metrics/MetricsManager.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+     *
+     * @param httpStatAttributes
```
Comment (fmhwong) [inline dev/com.ibm.ws.connectionpool.monitor/src/com/ibm/ws/connectionpool/monitor/metrics/MetricsManager.java]: The javadoc doesn't match the method parameters.

**Comment by Channyboy** [comment]
#build #libby

**Review by fmhwong — DISMISSED** [review]
I'd recommend to add a non-zero check for all metrics value before GA.

**Comment by Channyboy** [comment]
#build #libby

**Review by fmhwong — DISMISSED** [review]
LGTM

**Comment by Channyboy** [comment]
#build #libby

**Comment by Channyboy** [comment]
#build #libby

**Review by tonyreigns — APPROVED** [review]
LGTM

**Comment by Channyboy** [comment]
Failure and error seen in the telemetry TCK and telemetry logging are pre-existing issues.
Error seen in the monitoring FAT is due to test/machine slowness.

**Pull request merged**